EpikCord/__init__.py

```diff
@@ -973,7 +975,8 @@ async def ready(self, data: dict):
                         guild_id, commands
                     )
         return None
-
+    async def command_error(self, error:Exception, **kwargs):
+        raise error
```

Comment:
what's the point in extracting one single line into this method?

```suggestion
    async def command_error(self, error:Exception, **kwargs):
        raise error
```

Reply:
the point is that it is a customizable event which is triggered when command_event is called, and that's the default behavior of the event